src/tools/clippy/lintcheck/Cargo.toml

```diff
@@ -10,15 +10,15 @@ edition = "2018"
 publish = false
 
 [dependencies]
-clap = "2.33"
-flate2 = {version = "1.0.19"}
-fs_extra = {version = "1.2.0"}
-rayon = {version = "1.5.0"}
-serde = {version = "1.0", features = ["derive"]}
-serde_json = {version = "1.0"}
-tar = {version = "0.4.30"}
-toml = {version = "0.5"}
-ureq = {version = "2.0.0-rc3"}
+flate2 = "1.0"
+fs_extra = "1.2"
+rayon = "1.5"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+tar = "0.4"
+toml = "0.5"
+ureq = "2.2"
+walkdir = "2.3"
```

Comment:
This removes `clap` and adds `walkdir`?

Reply:
Removal of `clap` was me fat-fingering the patch by hand.

`walkdir` was added here: https://github.com/rust-lang/rust-clippy/commit/997ddbbfd8adf3a56307c817d562fc6d07bcbd6c

This version of clippy does yet not seem to include that commit.

Both problems were fixed with my most recent push.